kv/kv.go

```diff
@@ -456,10 +458,12 @@ type Driver interface {
 // Storage defines the interface for storage.
 // Isolation should be at least SI(SNAPSHOT ISOLATION)
 type Storage interface {
-	// Begin transaction
+	// Begin a global transaction
 	Begin() (Transaction, error)
-	// BeginWithStartTS begins transaction with startTS.
-	BeginWithStartTS(startTS uint64) (Transaction, error)
+	// Begin a transaction with the given txnScope (local or global)
+	BeginWithTxnScope(txnScope string) (Transaction, error)
+	// BeginWithStartTS begins transaction with given txnScope and startTS.
+	BeginWithStartTS(txnScope string, startTS uint64) (Transaction, error)
```

Comment:
I think the function name and recv parameter is unmatched here. Maybe we shall use a better name instead.

Reply:
I suggest `BeginWithStartTSInScope`, and change the order of the params.

Reply:
This is an interface definition that existed before, I think we should not change it easily. Not sure whether other components such as BR and tidb-lightning have used it. However, I think the name is reasonable here. Since the TSO now has the scope concept, i.e `txnScope`, when you provide this method a `startTS`, you should also point out which `txnScope` this TSO belongs to.